open-api/rest-catalog-open-api.yaml

```diff
@@ -1417,6 +1443,35 @@ paths:
         5XX:
           $ref: '#/components/responses/ServerErrorResponse'
 
+  /v1/aws/s3/sign:
```

Comment:
@nastra, I think it would be better to merge the specs _after_ this PR. There's already a lot changing and we don't need to move the other spec here at the same time.

Reply:
ok makes sense, I'll open a PR to merge the specs once this PR goes in